docs/communication_protocols/timer_based_protocol.rst

```diff
@@ -0,0 +1,147 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _timer_based_protocol:
+
+***************************************************
+Timer Based Protocols - PWM, DSHOT, and More
+***************************************************
+
+Timer based protocols (previously hobby protocols) refer to an array of different protocols commonly used by drone hobbyists for controlling ESCs. Vertiq modules support the following selection of the 
```

Comment:
Reading it again, we may want to lose the bit about "drone hobbyists". Might rob some of our customers who use these protocols the wrong way.

Maybe just "an array of different protocols commonly used for controlling ESCs"